polyphemus/lib/etls/metis_files_linker_base.rb

```diff
@@ -70,9 +96,18 @@ def files_payload(project_name:, model_name:, files:, attribute_regex:)
 
         next if payloads_for_attr.empty?
 
-        payload[attribute_name] = is_file_collection?(project_name, model_name, attribute_name) ?
-          payloads_for_attr :
-          payloads_for_attr.first
+        if is_file_collection?(project_name, model_name, attribute_name)
+          if record.nil? || (existing_files = record[attribute_name]).nil?
+            payload[attribute_name] = payloads_for_attr
+          else
+            payload_by_paths = payloads_for_attr.map { |f| f[:path] }
+            payload[attribute_name] = payloads_for_attr + existing_files.select do |file|
```

Comment:
Merges the new picked up files with any existing files.
When this reaches production, I'll reset the ETL and let it process existing datas again.  I'll be monitoring and improving performance overall.